test/.jenkins_tav_envs.yml

```diff
@@ -0,0 +1,5 @@
+ENVS:
```

Comment:
This is probably just my lacking knowledge of Jenkins shining though, but I'm not sure if `ENVS` means something in Jenkins-terms? If not, I think this would be better named `TAV` as that is the name of [the associated environment variable](https://github.com/watson/test-all-versions#whitelist-tests-with-environment-variables) these values will be used for.

If on the other hand `ENVS` this makes perfect sense for a person more familiar with Jenkins, then just ignore this comment 😄

Reply:
I've renamed it to `TAV`. 
The naming was not Jenkins specific but aligned with how it was called in the `.travis.yml`: `env: TAV=xx`.